pyiron_atomistics/atomistics/structure/factory.py

```diff
@@ -332,3 +332,65 @@ def element(parent_element, new_element_name=None, spin=None, potential_file=Non
                 parent_element=parent_element, new_element=new_element_name
             )
         return periodic_table.element(new_element_name)
+
+# importing the modules required by aimsgb
+from pyiron.atomistics.structure.atoms import pyiron_to_pymatgen, pymatgen_to_pyiron
+from pymatgen import Structure, Lattice, PeriodicSite
+from aimsgb import GrainBoundary, Grain, GBInformation
+
+class GBBuilder:
+
+    def gb_info(axis, max_sigma):
+        """
+        Provides a list of possible GB structures for a given rotational axis and upto the given maximum sigma value.
+
+        Args:
+            axis : Rotational axis for the GB you want to construct (for example, axis=[1,0,0])
+            max_sigma (int) : The maximum value of sigma upto which you want to consider for your
+            GB (for example, sigma=5)
+
+        Returns:
+            A list of possible GB structures.
+        """
+        print("Possible GB structures for rotational axis",axis,"and sigma cutoff", max_sigma,"are listed in the format:\n")
+        print("{sigma value: {'theta': [theta value],")
+        print("'plane': the GB plane")
+        print("'rot_matrix': the rotational matrix,")
+        print("'csl': the csl matrix}}\n")
+        print("To construct the grain boundary select a GB plane and sigma value from the list below and \
+        \npass it to the GBBuilder.gb_build() function along with the rotational axis and initial bulk structure.")
+
+        return GBInformation(axis=axis, max_sigma=max_sigma)
+
+    def gb_build(axis, sigma, plane, initial_struct, to_primitive=False,
```

Comment:
```suggestion
    @staticmethod
    def gb_build(axis, sigma, plane, initial_struct, to_primitive=False,
```

Reply:
Thank you for the suggestions. I will modify the code accordingly. I am not sure about the class. Should I remove it or keep it. I thought may be we can add more things to the GBBuilder class in the future